server/src/main/java/org/opensearch/common/blobstore/stream/write/WriteContext.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.common.blobstore.stream.write;
+
+import org.opensearch.common.Nullable;
+import org.opensearch.common.StreamContext;
+
+import java.util.function.Consumer;
+
+/**
+ * WriteContext is used to encapsulate all data needed by <code>BlobContainer#writeStreams</code>
+ *
+ * @opensearch.internal
+ */
+public class WriteContext {
```

Comment:
Lets add tests for WriteContext

Reply:
This is just a model class, testing for this is covered by the tests in #7119. Do we need separate tests here?